src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -55,16 +54,72 @@ export default class QuestionnaireForm extends Component {
     }
 
     const items = this.props.qform.item;
-    this.prepopulate(items, newResponse.item, saved_response)
+    this.prepopulate(items, newResponse.item, false)
+    this.state.savedResponse = newResponse;
+  }
+
+  componentDidMount() {
+    this.loadAndMergeForms(this.state.savedResponse);
+  }
+
+  loadPreviousForm() {
+    // search for any QuestionnaireResponses 
+    this.smart.request("QuestionnaireResponse?" + 
+          "&author=" + this.getPractitioner() + 
```

Comment:
should this really be requesting ALL responses, instead of just in-progress ones.

Reply:
Yes, we want all of the forms here.